### DISCUSSION TITLE
Docker Image: is the libreoffice component already included?

### DISCUSSION THREAD

**Discussion by jotpunktopunkt** [Stirling-Tools/Stirling-PDF#696] [Q&A] [answered]
hey

thank you for this great tool!
i'm new to docker etc and can't make out wether the libreoffice components are already installed when using the docker-compose from the github page

do i have to additionally install them?

**Comment by Frooodle** [comment, marked as answer]
already installed :)

**Reply by jotpunktopunkt** [reply]
thank you very much :)
